Bottom: 2dc0413d723630f43ba512d176eca5f287838dc6
Top:    c7ef9210846135cbbe6f1a210b0ad0c08bb9fa82
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:40:57 +0000

Refresh of fix-3d-contour-export

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 2352729..b8bd25c 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -266,7 +266,7 @@ get_contour_edge_n_round_geometry_in_step_mm (PLINE *contour, int n, double *cx,
       *cx = COORD_TO_STEP_X (PCB, contour->cx);
       *cy = COORD_TO_STEP_Y (PCB, contour->cy);
       *r = COORD_TO_MM (contour->radius);
-      *cw = (contour->Flags.orient != PLF_DIR);
+      *cw = (contour->Flags.orient == PLF_INV);
     }
 #endif
 }
